rollup.config.js

```diff
@@ -1,5 +1,6 @@
 import typescript from "@rollup/plugin-typescript"
 import commonjs from "@rollup/plugin-commonjs"
+import nodeResolve from "@rollup/plugin-node-resolve"
```

Comment:
I noticed this in the dependency changes too, what is it for?

Reply:
This is to pull in the js-joda library into the zealot bundle. We previously had no external dependencies.